### DISCUSSION TITLE
Flat-styled Windows logos (from 1 to 11) start buttons

### DISCUSSION THREAD

**Discussion by ghost** [Open-Shell/Open-Shell-Menu#1879] [Show and tell]
Hi,
this is a small collection of start buttons that I made, meant to be used as a custom start menu button in Open-Shell.

![preview](https://github.com/user-attachments/assets/a7d9cf58-d1d4-4181-8c0f-f1dff89e49e3)

It features every Microsoft Windows logo so far, from 1 to 11. I made them transparent, for both dark and white themes. Doesn't clash with custom theme colors.

Looks best on Windows 10.

Feel free to use them for whatever else. Microsoft Windows logo design is made by Microsoft, obv. :-)

[fswlsb.zip](https://github.com/user-attachments/files/16644219/fswlsb.zip)

**New:** I made a variant featuring the start text right next to the logo, making it like the Windows XP start button.

**Comment by lnpeters** [comment]
These look soooo good! Thanks!

**Reply by ghost** [reply]
Thank you!

**Comment by ghost** [comment]
**New:** I made a variant featuring the start text right next to the logo, making it like the Windows XP start button. Check it out! ;-)

**Comment by haikolojik** [comment]
Yükleme nasıl acaba?

**Comment by mea2008** [comment]
I have never commented on Github, but I really really liked what you did here. Thank you :) 
![Screenshot 2025-06-15 170946](https://github.com/user-attachments/assets/893dd71c-6315-43eb-ad9b-d3ca27d10eab)

my start menu looks so good now

**Reply by mea2008** [reply]
one of the main things is that your start menu button centered perfectly

**Comment by SteelBodyX** [comment]
<img width="236" height="151" alt="image" src="https://github.com/user-attachments/assets/d67f4fe9-9c5e-4d26-bee8-c7ae2aa5f44e" />
Is it possible to get these in colors? Like it can be greyscale but mouse hovers over it, the icon becomes colorful. I'm rocking the Vista icon for a long time and would love to see a flatter and more colorful icon.

**Comment by pacomi123456789-a11y** [comment]
Can you make different versions for different display sizes?

**Comment by bluefour18-afk** [comment]
I want one that looks like this, https://gyazo.com/8b5726a347316d092c2eb4316e30b30d 
The one I have doesn't seem to work. When I try to select it I am able to put the file path in but it will not show and just keeps the shell icon. I think it is because I put the image on an external drive when I updated to windows 11 from windows 10 (which had classic shell on it and that is what the image worked on with it). So if someone could use the look of above but in the correct format for open shell, it would be much appreciated.

**Comment by dpi0** [comment]
🗿

**Comment by bluefour18-afk** [comment]
<img width="185" height="250" alt="Start buttons" src="https://github.com/user-attachments/assets/22f6ee7f-9518-4523-81f7-952af5d3a79c" />
 Does anyone have start buttons like this in the correct format for open shell?

**Reply by FreshKorre2001** [reply]
That three split right there confuses me, how can one do that with GIMP? I wanted to do this with an N64 logo that I will be using as my Start Button
